### ISSUE TITLE
Questions about the code of adversarial training 

### ISSUE THREAD

**Issue by Link-world** [qiufan319/benchmark_pc_attack#2]
Hi, thanks for your benchmark work and the open source code. I have a question about the AT part of the code. 

In your paper, the section on adversarial training references the paper 'Extending adversarial attacks and defenses to deep 3D point cloud classifiers.', in which adversarial examples are progressively updated during the model training process and original point clouds are used both for clean training and for generating adversarial examples. 

However, in the AT part of the code, the adversarial examples are fixed, and the adversarial and clean examples come from different original point cloud data. I would like to know which method are used for adversarial training in the benchmark leaderboard and if the two methods will lead to different defense effect. 

I‘d appreciate it if you could answer my question. Thanks again for your work.

**Comment by qiufan319** [comment]
Hi @Link-world. You are right. In my benchmark, I used fixed adversarial training, and the same method is also used in hybrid training. Using progressively updated adversarial training would indeed make the model training more robust, but due to time constraints, I did not use progressively updated adversarial training in my benchmark. You are welcome to try this training method to improve and comprehensively enhance the benchmark.
